app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
You can leave only Player here
(abstract class must have at least one abstract method, class Dwarf - not)